src/App.js

```diff
@@ -0,0 +1,16 @@
+import "./App.css";
+import { Filters } from "./components/productListing/Filters";
+import { ProductList } from "./components/productListing/ProductList";
+// import { LandingPage } from "./pages/LandingPage";
+import LoginPage from "./pages/LoginPage";
+import PasswordResetPage from "./pages/PasswordResetPage";
+import SignupPage from "./pages/SignupPage";
```

Comment:
Use index.js file pages and component folders to export your components. 
`Why?` This will make your code cleaner.
`Note` If you're planning to implement this then you can ignore this comment. 
For additional info read about `Barrel Export In React.`

Reply:
Yes I am planning to implement this in further PRs when I will add the functionality.